inference-engine/tests/unit/multi/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
@nosovmik , you recently has implemenetd the mock-based tests for the caching, can you pls take a look on this PR from general sanity, e.g. whether we need the dedicated target, etc?